pages/clubs-list.jsx

```diff
@@ -0,0 +1,237 @@
+var _ = require('underscore');
+var React = require('react');
+var ModalManagerMixin = require('../mixins/modal-manager');
+var TeachAPIClientMixin = require('../mixins/teach-api-client');
+var HeroUnit = require('../components/hero-unit.jsx');
+var Map = require('../components/map.jsx');
+var ModalAddOrChangeYourClub = require('../components/modal-clubs.jsx');
+var ModalRemoveYourClub = require('../components/modal-clubs-remove.jsx');
+
+var ClubListItem = React.createClass({
+  propTypes: {
+    club: React.PropTypes.object.isRequired,
+    username: React.PropTypes.string,
+    onDelete: React.PropTypes.func,
+    onEdit: React.PropTypes.func,
+    onZoomToLocation: React.PropTypes.func.isRequired
+  },
+  handleLocationClick: function() {
+    this.props.onZoomToLocation(this.props.club);
+  },
+  render: function() {
+    var club = this.props.club;
```

Comment:
probably worth pulling out `this.props` since we refer to `this.props.[...]` in a number of places here